test_defopt.py

```diff
@@ -912,6 +931,12 @@ def test_choices_cli(self):
         self.assertIn(b'four', error.exception.output)
         self.assertIn(b'{one,two,three}', error.exception.output)
 
+    def test_exceptions(self):
+        self._run_example(exceptions, ['1'])
+        with self.assertRaises(subprocess.CalledProcessError) as error:
+            self._run_example(exceptions, ['0'])
+        self.assertIn(b"Don't do this!", error.exception.output)
```

Comment:
Could also check that `"Traceback (most recent call last)"` or similar doesn't appear in the output.

Reply:
good point, done